src/object/srv_ec_aggregate.c

```diff
@@ -947,8 +948,8 @@ agg_fetch_remote_parity(struct ec_agg_entry *entry)
 		if (i == pidx)
 			continue;
 		d_iov_set(&iov, &buf[i * cell_b], cell_b);
-		peer_shard = obj_ec_parity_shard(entry->ae_dkey_hash,
-						 &entry->ae_oca,
+		peer_shard = obj_ec_parity_shard(obj_hdl2ptr(entry->ae_obj_hdl),
```

Comment:
here too, please check all other places of obj_hdl2ptr() in this patch, I won't point out again to avoid too many, thx

Reply:
ah, sure. thanks.